README.md

```diff
@@ -13,4 +13,4 @@ This is from oniontest2. But commit shows it's from oniontest1... Be careful.
 
 Edit from oniontest1:
 ---
-I'd suggest practicing git more.
\ No newline at end of file
+I'd suggest practicing git using multiple accounts.
```

Comment:
@oniontest1 #1 Maybe elaborate a little bit more?

Reply:
`OK`, I'll do it.